examples/storage/s3/incremental_learning/README.md

```diff
@@ -15,19 +20,15 @@ metadata:
 stringData: # use `stringData` for raw credential string or `data` for base64 encoded string
   ACCESS_KEY_ID: XXXX
   SECRET_ACCESS_KEY: XXXXXXXX
+EOF
 ```
 
-### Attach the created secret to the Model/Dataset/Job.
-`EDGE_NODE` and `CLOUD_NODE` are custom nodes, you can fill it which you actually run.  

```

Comment:
"you can fill it which you actually run.  " I don't understand what is that means

Reply:
this file has been deleted